libs/ui/src/lib/textarea/textarea.component.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <ng-container *ngIf="label !== ''">
```

Comment:
What if the label is `undefined` or `null`??
We will still render an empty label
This will also cover those cases along with the `''`
```suggestion
    <ng-container *ngIf="label">
```